prototype1/src/app_net/helpers.rs

```diff
@@ -11,7 +11,9 @@ mod helpers {
       .and_then(|external_ip| TcpStream::connect((external_ip, 80)))
       .and_then(|stream| {
         let addr = stream.local_addr();
-        stream.shutdown(Shutdown::Both);
+
+        // We don't care about the tcp connection, just the IP
+        let _ = stream.shutdown(Shutdown::Both);
```

Comment:
I want to know the idiomatic way to squelch these errors when we really don't care about the error. `stream.shutdown(Shutdown::Both).unwrap_or(());`? Annotate with 'allow this warning'? `let _` is just whack